### PR TITLE
Update release_notes.md

### PR DESCRIPTION
**Pull request by lizygeogy**
@rohmurthy 
@venkateshkottapalli 
@sandeshh 
@patilvikram 

Please review and provide feedback for the 3.9.2 releases notes.

### CODE REVIEW

**Comment by rohmurthy** [comment]
@sandeshh  I have reviewed, once you do a quick sanity check, we can merge this

File: docs/release_notes.md [hunk 1]
```diff
@@ -5,14 +5,33 @@ Version: 3.9.2
...
 
-* [SPOI-12791] - With anti affinity STRAM not getting enough containers from RM to run all the operators.
-* [SPOI-12794] - Files configured by the user are not available when upgraded.
-* [SPOI-12795] - Metrics platform not supported with DT_Plus license.
-* [SPOI-12832] - Application exiting because of NPE in application master.
-* [SPOI-12855] - Update CassandraStore to support more than 1 cassandra nodes.
+**[SPOI-12791]**
+
+In the application when the anti-affinity is enabled, the application master gets containers from the Resource Manager for sometime only.  After this no more containers are received even after periodically requesting the STRAM for containers. Thus the functioning of the application gets disrupted.
+This issue is now resolved.
```
Comment (sandeshh) [inline docs/release_notes.md]: Why do we need "This issue is now resolved" after every point, if the whole thing is already under "Resolved issues" ?

**Comment by sandeshh** [comment]
@rohmurthy I have given one comment on the PR. Another question is, do we not mention about the UI bug fixes?

**Comment by rohmurthy** [comment]
@sandeshh Yes all bugs reports by customers should be incorporated in Release Notes

**Comment by venkateshkottapalli** [comment]
FYI, To keep track, it doesn't provide the details of  SPOI-12932 which is reported Friday PST morning time and included in 3.9.2.

**Comment by lizygeogy** [comment]
@rohmurthy 
@sandeshh 
@venkateshkottapalli 
@patilvikram 
All the fixed issues as suggested by Rohini are added to the release notes here. Please review and provide your valuable feedback at the earliest.

**Comment by rohmurthy** [comment]
Reviewed, going to approve this

**Pull request merged**